README.md

```diff
@@ -1,2 +1,34 @@
-# matt-item-summary
-item summary module
+# Wawa Melon Item Summary
+Item Summary Module
+
+## Related Projects
+
+  - https://github.com/fec3-galadriel/Review-component
+  - https://github.com/fec3-galadriel/mike-photo-carousel
+  - https://github.com/fec3-galadriel/garrett-related-products
+
+## Table of Contents
+
+1. [Usage](#Usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+
+## Usage
+
+> Some usage instructions
+## Requirements
+
+An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+
+- Node 6.13.0
+- etc
+
+## Development
+
+### Installing Dependencies
+From within the root directory:
+
+```sh
+npm install webpack
```

Comment:
Looks like you have webpack listed as a devDependency - `npm install` should be enough.

Reply:
Good call! I need to update the README again, adding it to my Trello board :)